pandas/tests/frame/test_indexing.py

```diff
@@ -273,6 +273,8 @@ def test_getitem_boolean(self):
         # test df[df > 0]
         for df in [self.tsframe, self.mixed_frame,
                    self.mixed_float, self.mixed_int]:
+            if compat.PY3 and df is self.mixed_frame:
+                continue
```

Comment:
let's strip out the mixed_frame to another function (even though that duplicates some code), bonus can parametrize this test.

Reply:
> bonus can parametrize this test.

I don't think `tsframe`, `mixed_frame`, `mixed_float`, `mixed_int` are available in the namespace.

Reply:
these need to be made fixtures. this becomes so much easier.

Reply:
I agree, am starting to implement this in the test_arithmetic sequence of PRs.  Will update this test when that lands.

Reply:
k thanks